### PR TITLE
clean up old files

### PR DESCRIPTION
**Pull request by Cr0wn-Gh0ul**
## Scope

What is changing with this PR?
Deleting old unused files

## Why?
They are no longer needed

Why are we doing this?
To not have clutter

## Todos
N/A
If any, what are the follow-up tasks required other than merging this PR? Have they been arranged?

- [ ]

### CODE REVIEW

File: assets/test/keys/password.txt [hunk 1]
```diff
@@ -1 +0,0 @@
-abc123
```
Comment (diligentis) [inline assets/test/keys/password.txt:1]: This file is still referenced in `scripts/base-scripts/geth-local-resume.sh`

File: assets/test/keys/UTC--2020-03-24T13-41-09.685691521Z--546f99f244b7b58b855330ae0e2bc1b30b41302f [hunk 2]
```diff
@@ -1 +0,0 @@
-{"address":"546f99f244b7b58b855330ae0e2bc1b30b41302f","crypto":{"cipher":"aes-128-ctr","ciphertext":"b90487f543b540ae3a3389005348e04d9c5220221cda52e7364196603f942b49","cipherparams":{"iv":"bd812cee655ff3c6e5eefc37d3c4f7d0"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"7d38390aa3339425df6766cd4b58ba4719d272612653f0ec75ba65f1e90c3c54"},"mac":"de3155975dd570eed3af001c805f614056b77251efb0926bc9013a3db9c35ebb"},"id":"6b2a0716-b444-46c3-a1e3-2936ddd8ecc5","version":3}
```
Comment (diligentis) [inline assets/test/keys/UTC--2020-03-24T13-41-09.685691521Z--546f99f244b7b58b855330ae0e2bc1b30b41302f:1]: These files are still referenced by `scripts/base-scripts/geth-local.sh`

File: assets/config/validator0.toml [hunk 3]
```diff
@@ -1,62 +0,0 @@
-[loglevel]
```
Comment (diligentis) [inline assets/config/validator0.toml:1]: Files still referenced by `docker-compose.yml` and `scripts/main.sh`.

File: bridge/deployments/dev/deploymentArgsTemplate.json [hunk 4]
```diff
@@ -1,62 +0,0 @@
-{
```
Comment (diligentis) [inline bridge/deployments/dev/deploymentArgsTemplate.json:1]: File still referenced in `README.md`

**Review by diligentis — CHANGES_REQUESTED** [review]
A few references to clean up

File: assets/config/validator0.toml (hunk 3 above)
Comment (Cr0wn-Gh0ul) [reply assets/config/validator0.toml:1]: docker-compose is all kinds of broken

**Comment by Cr0wn-Gh0ul** [comment]
Found one more ref in some really old file. Deleted that as well.

**Comment by Cr0wn-Gh0ul** [comment]
Well then, that terrible file was for an outdated test, I will correct that.

**Comment by Cr0wn-Gh0ul** [comment]
eventually i will stop dropping the ball on this

**Review by diligentis — APPROVED** [review]
LGTM

**Comment by Cr0wn-Gh0ul** [comment]
Lost some code coverage on the config test, will have to revisit that when working on the config parser to remove remaining old config syntax

**Review by nelsonhp — APPROVED** [review]

**Pull request merged**